release/README.md

```diff
@@ -1,93 +1,37 @@
-docker-compose logs -f
-```
+GitHub Actions will perform the release process. Please take note of the workflow run number. You will find this in the text "Release Artifacts #<worflow run number>"
 
-Follow specific container logs:
-```
-docker-compose logs -f data-prepper
-```
+Wait for the job to complete successfully.
 
-The Data Prepper container should show events being processed:
-```
-...
-data-prepper_1         | 2021-12-03T19:57:30,488 [service-map-pipeline-prepper-worker-3-thread-1] INFO  com.amazon.dataprepper.pipeline.ProcessWorker -  service-map-pipeline Worker: No records received from buffer
-data-prepper_1         | 2021-12-03T19:57:31,925 [grok-pipeline-prepper-worker-7-thread-1] INFO  com.amazon.dataprepper.pipeline.ProcessWorker -  grok-pipeline Worker: Processing 8 records from buffer
-...
-```
-### 3. Confirm Open Search is running
-```
-curl -s -k -u 'admin:admin' 'https://localhost:9200/_cat/indices'
-```
-If indicies are displayed Open Search is running.
-```
-yellow open security-auditlog-2021.12.03   8FGpNXmYRamkbuOsdTRORQ 1 1 13 0 75.1kb 75.1kb
-yellow open test-grok-index                WRuIkpsiQXOcGirpp9hRqw 1 1 77 0 39.1kb 39.1kb
-yellow open .opendistro-job-scheduler-lock cyeA9SeST5ibZQPW24CJPA 1 1  1 1   11kb   11kb
-yellow open otel-v1-apm-service-map        uHdBor9FTSiJDwLAHLroVw 1 1  0 0   208b   208b
-green  open .opendistro_security           IAWYPcinRz66_mamfcLsew 1 0  9 0 57.8kb 57.8kb
-yellow open otel-v1-apm-span-000001        3pN9V1YNSjSMY5ggHa2TfA 1 1 12 0 28.3kb 28.3kb
-```
-**To confirm if Open Search is receiving log data run**
-Use the following cURL command to query the index `test-grok-index` for any documents. **Note**: The results may not show all documents received.
-```
-curl -k -u 'admin:admin' https://localhost:9200/test-grok-index/_search
-```
-If in your results the JSON path `.hits.total.value` has a value of 0 Open Search is not receiving log data. Confirm with the Data Prepper logs records are being processed from the buffer and no error messages are displayed.
+### Promotion to Production
 
-**To confirm if Open Search is receiving trace data run**
-Use the following cURL command to query the `otel-v1-apm-span-000001` index for documents containing **PythonService**. These documents will be generated by the **otel-span-exporter**, sent to **otel-collector**, then sent to **data-prepper**, then sent to **opensearch**.
-```
-curl -k -u 'admin:admin' https://localhost:9200/otel-v1-apm-span-000001/_search?q=PythonService
-```
-If in your results the JSON path `.hits.total.value` has a value of 0 Open Search is not receiving trace data. Confirm OTel opentelemetry-collector logs are continueally displaying metrics and no errors are printing. Next confirm with the Data Prepper logs records are being processed from the buffer and no error messages are displayed.
+Once you have a completed build, you may update the release Issue you created in the opensearch-build project. Provide the following:
```

Comment:
It might be helpful to explain what the following info is used for. Is it for opensearch-build to pick up the right artifacts in S3 and docker image in ECR?